Service/GetPaymentSessionData.php

```diff
@@ -139,4 +154,18 @@ public function execute(string $orderId, ?array $customData = null): array
 
         return $payload;
     }
+
+    protected function getAvailablePaymentMethods() : array {
```

Comment:
Is this intended to be card specific? Or should Pay by Bank be an option in here?

- [ ] PBB as available payment method?

Reply:
This is card specific, pay by bank is a completely separate payment method while card component facilitates both express options and card.